src/states_screens/tracks_screen.cpp

```diff
@@ -142,7 +142,7 @@ void TracksScreen::beforeAddingWidget()
     if (group_amount > 1)
     {
         //I18N: name of the tab that will show tracks from all groups
-        tabs->addTextChild( _("All"), ALL_TRACK_GROUPS_ID );
+        tabs->addTextChild( _(ALL_TRACK_GROUPS_ID), ALL_TRACK_GROUPS_ID );
```

Comment:
The "all" track group is kind of special, since there is actually no such group. It's just a way of saying "any group". How would you do it?